From 1db4d34759991cf571eabe4f2d013f336af1979b Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:18 -0400
Subject: [redhat] kabi: Adding symbol __request_module (kernel/kmod.c)

Message-id: <9acaea9c915d2002f00f46b604257f43e03cb428.1410510773.git.poros@redhat.com>
Patchwork-id: 95979
O-Subject: [PATCH RHEL7.1 018/233] kabi: Adding symbol __request_module (kernel/kmod.c)
Bugzilla: 1071097
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Try to load a kernel module.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/__request_module
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/__request_module
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/__request_module

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/__request_module b/redhat/kabi/kabi-module/kabi_ppc64/__request_module
new file mode 100644
index 0000000..50f9dd8
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/__request_module
@@ -0,0 +1,2 @@
+#1-
+0x402b8281 __request_module vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/__request_module b/redhat/kabi/kabi-module/kabi_s390x/__request_module
new file mode 100644
index 0000000..50f9dd8
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/__request_module
@@ -0,0 +1,2 @@
+#1-
+0x402b8281 __request_module vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__request_module b/redhat/kabi/kabi-module/kabi_x86_64/__request_module
new file mode 100644
index 0000000..50f9dd8
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__request_module
@@ -0,0 +1,2 @@
+#1-
+0x402b8281 __request_module vmlinux EXPORT_SYMBOL
-- 
1.7.1